src/main/java/org/jetbrains/skija/Matrix44.java

```diff
@@ -0,0 +1,22 @@
+package org.jetbrains.skija;
+
+import lombok.Data;
+import org.jetbrains.annotations.*;
+
+@Data
+public class Matrix44 {
+    @ApiStatus.Internal
+    public final float[] _mat;
+
+    public Matrix44(float... mat) {
+        assert mat.length == 16 : "Expected 16 elements, got " + mat == null ? null : mat.length;
```

Comment:
What if we run without `-ea`? Better throw `InvalidArgumentException`, IMO.

Reply:
I copy-pasted Matrix33 :).

assert is used in all of skija.

@tonsky, do you have opinion - what should be used: "assert" or exceptions?
In java "assert" is prettier in my opinion :), but don't work without -ea, so I would also choose exceptions.

Reply:
That’s the idea. You test your app with -ea, then run it without it, not paying for asserts